app/src/main/java/com/uncookthebook/app/ReportArticleFragment.java

```diff
@@ -47,4 +47,9 @@ private void getURL() {
             Log.d("URL", url);
         }
     }
+
+    @Override
+    public void onClick(View v) {
+
```

Comment:
Maybe write a comment that explains why the body of the method is null but we have to override it anyway. It is a bit unclear to me.

Reply:
oh it's going to get overritten it's just that I didn't do that yet. It's for the reportArticleFragment in which we will have to add all the buttons